xarray/computation/ops.py

```diff
@@ -158,8 +161,8 @@ def fillna(data, other, join="left", dataset_join="left"):
     )
 
 
-# Unsure why we get a mypy error here
-def where_method(self, cond, other=dtypes.NA):  # type: ignore[has-type]
+# TODO: type this properly
+def where_method(self: Any, cond: Any, other: Any = dtypes.NA):
```

Comment:
```suggestion
def where_method(self, cond, other = dtypes.NA):  # type: ignore[has-type]
```

Reply:
Not sure about the type failure, so we can just ignore it for now.

Reply:
> Let's just ignore it and merge

Ah ok I misunderstood then.  I thought we could just ignore it and then `mypy` would stop complaining going forward in other PRs.  

Do you have any suggestions for how I could address this issue?

Reply:
By "ignore" i meant `type: ignore` hehe. Sorry for the confusion. We do need that CI test to be green so just make it work :D